manifests/params.pp

```diff
@@ -1,11 +1,24 @@
 # PHP params class
 #
+# === Parameters
+#
+# [*config_root_override*]
+#   The configuration root directory used in parameter calculation
+#
+# [*php_version_override*]
+#   The version of php used in parameter calculation
+#
 class php::params(
-  $cfg_root = undef, # lint:ignore:parameter_documentation
+  $config_root_override = undef,
+  $php_version_override = undef
```

Comment:
I didn't think anyone used puppet without hiera but OK. Shall I just leave the params as they were? Or remove `cfg_root`, `cfg_root` and `php_version_override`?